ubuntu-core-24-amd64-dangerous.json

```diff
@@ -32,6 +32,13 @@
             "type": "snapd",
             "default-channel": "latest/edge",
             "id": "PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4"
+        },
+        {
+            "name": "console-conf",
+            "type": "app",
+            "default-channel": "latest/edge",
```

Comment:
we could use a `24` track for console-conf

Reply:
will update these soon with new models referring to 24 track instead